packages/wallet/contract/src/walletFactory.js

```diff
@@ -10,22 +10,32 @@ import '@agoric/zoe/exported.js';
 
 import { makeAtomicProvider } from '@agoric/store/src/stores/store-utils.js';
 import { makeScalarBigMapStore } from '@agoric/vat-data';
-import { Far } from '@endo/far';
-import { makeSmartWallet } from './smartWallet.js';
+import * as BRIDGE_ID from '@agoric/vats/src/bridge-ids.js';
+import { E, Far } from '@endo/far';
+import { makeSmartWallet } from './smartWallet';
 
 /**
  * @typedef {{
  *   agoricNames: NameHub,
  *   board: Board,
  *   namesByAddress: NameHub,
  * }} SmartWalletContractTerms
+ *
+ * @typedef {{
+ * 	 type: string, // WALLET_ACTION
```

Comment:
is the type `ActionType`? 
```suggestion
 * type: keyof ActionType
```

Reply:
yes, but `ActionType` is written as a module namespace, not a record, so `keyof` doesn't work.

I have a refinement pending.